test/views/PaymentAllocationViewSpec.scala

```diff
@@ -423,7 +423,7 @@ class PaymentAllocationViewSpec extends ViewSpec with ImplicitDateFormatter {
 
     "not have Credit on account row within payment details" in new PaymentAllocationSetup(outstandingAmount =
       paymentAllocationViewModelWithCreditZeroOutstandingOutstandingAmount) {
-      document.getElementById("money-on-account") shouldBe null
+      Option(document.getElementById("money-on-account")).isDefined shouldBe false
```

Comment:
I would suggest to create next method instead of document.getElementById
`
findElementById(id: String) : Boolean = {
  Option(document.getElementById("money-on-account")).isDefined
}
`